Releasing.md

```diff
@@ -160,14 +156,6 @@ For the body of the post, just copy this checklist and again replace all occurre
 <p>o If there are any conflicts in the Gutenberg PR, merge <code>trunk</code> into it and resolve them.</p>
 <!-- /wp:paragraph -->
 
-<!-- wp:paragraph -->
-<p>o Check if you can open a PR from <code>trunk</code> to <code>develop</code> in Gutenberg Mobile without any conflicts: https://github.com/wordpress-mobile/gutenberg-mobile/compare/develop...trunk. If there are any conflicts, create a branch from <code>trunk</code> with a name like <code>merge_release_x.xx.x_to_develop</code>, merge <code>develop</code> into it, resolve any conflicts.</p>
-<!-- /wp:paragraph -->
-
-<!-- wp:paragraph -->
-<p>o Open a PR from Gutenberg Mobile <code>trunk</code> (or <code>merge_release_x.xx.x_to_develop</code> branch) to <code>develop</code>.</p>
-<!-- /wp:paragraph -->
-
 <!-- wp:paragraph -->
 <p>o Merge the Gutenberg PR to <code>trunk</code>.</p>
```

Comment:
This is not your change, but it looks like there are two `Merge the Gutenberg PR to <code>trunk</code>` steps (here and 8 lines below). I'm thinking the second one should be `Gutenberg-Mobile` instead of `Gutenberg`.